python/ray/includes/libcoreworker.pxd

```diff
@@ -87,14 +88,15 @@ cdef extern from "ray/core_worker/core_worker.h" nogil:
             const CRayFunction &function,
             const c_vector[unique_ptr[CTaskArg]] &args,
             const CTaskOptions &options, c_vector[CObjectID] *return_ids,
-            int max_retries)
+            int max_retries,
+            c_pair[CPlacementGroupID, int64_t] placement_options)
```

Comment:
I thought about moving this into CTaskOptions, but since it only makes sense for non-actor tasks, it's cleaner to keep as a separate arg specifically for SubmitTask.